tests/e2e/default/enketo/pregnancy-visit.wdio-spec.js

```diff
@@ -37,7 +37,7 @@ describe('Pregnancy Visit', () => {
 
     //report details
     const openReportInfo = await reportsPage.getOpenReportInfo();
-    expect(openReportInfo.senderName).to.equal(`Submitted by ${userData.userContactDoc.name} `);
+    expect(openReportInfo.senderName).to.equal(`Submitted by ${userData.userContactDoc.name}`);
```

Comment:
Does this version automatically trim? I seem to remember a commit with .trim recently...

Reply:
I think it does something along those lines. Something is happening with spaces, because there were several changes around how spaces were treated that I need to make.